app/components/base/LangSwitcher.vue

```diff
@@ -0,0 +1,25 @@
+<script lang="ts" setup>
```

Comment:
I don't see any problem with your code.
But please run the `yarn lintfix` before pushing any commit.
It would be great if we installed `husky` to automatically run `yarn lintfix` before committing anything locally.
https://www.npmjs.com/package/husky